### PR TITLE
Update package version for DAOS API major version bump

### PR DESCRIPTION
**Pull request by kccain**
And update packaging files from latest packaging repo master.

PR-repos: daos@PR-3935:32
PR-repos: mpich@PR-44

Signed-off-by: Kenneth Cain <kenneth.c.cain@intel.com>

### CODE REVIEW

**Comment by brianjmurrell** [comment]
Update the `packaging/` subdir before trying to go any further.

File: packaging/Dockerfile.centos.7 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2019, Intel Corporation
+# Copyright 2018-2020, Intel Corporation
```
Comment (brianjmurrell) [inline packaging/Dockerfile.centos.7:2]:
```suggestion
# Copyright 2018-2021, Intel Corporation
```
now.

File: packaging/Dockerfile.ubuntu.20.04 [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2019-2020, Intel Corporation
```
Comment (brianjmurrell) [inline packaging/Dockerfile.ubuntu.20.04:2]:
```suggestion
# Copyright 2019-2021, Intel Corporation
```

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Is this another package that doesn't have a direct dependency on `libdaos,so` and therefore unaffected by a version bump in it?

**Comment by kccain** [comment]
Does the fact that it depends on mpich (that itself has changed due to a dependency on the new libdaos API) have relevance? And then that it provides a package that daos (Jenkinsfile) requires when it provisions packages on nodes for testing?

**Comment by brianjmurrell** [comment]
> Does the fact that it depends on mpich (that itself has changed due to a dependency on the new libdaos API) have relevance?

Only if the ABI of mpich changed, which I don't think it did, did it?

> And then that it provides a package that daos (Jenkinsfile) requires when it provisions packages on nodes for testing?

So this is a provides/requires that probably ought not exist.  Frankly none of the virtual provides/requires probably needs to exist for anything.  RPM automatically detects library (including) version dependencies.  I was in fact going to experiment with getting rid of them altogether at some future date and had it in my mind to have you get rid of them in your PRs but didn't want to burden your effort with that.

But if this tool doesn't actually need to be rebuilt, we should get rid of the virtual provides/requires for it and just use the actual package name as the dependency in the DAOS `Jenkinsfile`.

**Comment by kccain** [comment]
This PR is not needed, closing.

**Pull request closed without merging**